core/lib/src/fs/rewrite.rs

```diff
@@ -215,12 +221,24 @@ impl<'r> From<File<'r>> for Rewrite<'r> {
     }
 }
 
+impl<'r> From<File<'r>> for Option<Rewrite<'r>> {
+    fn from(value: File<'r>) -> Self {
+        Some(Rewrite::File(value))
+    }
+}
```

Comment:
What's the utility of this `impl`? I don't see it used anywhere, and I'm not sure we really want to hide the functionality under an opaque `.into()`.

Reply:
The other impl (`From<Redirect>` for `Option<Rewrite>`) is actually used in at least one doc-test.

This makes it possible for a rewrite to return `File { .. }.into()`, rather than `Some(File { .. }.into())`. I think it makes the API more succinct, without losing any information. I think it might be worth looking at the other doc-tests, and using this impl if appropriate.

Reply:
When adding documentation, now both are used (in the doc-tests for `FileServer::filter` and `FileServer::map` respectively).